docs/querying/sql-functions.md

```diff
@@ -136,6 +136,42 @@ Returns an array of all values of the specified expression.
 
 Concatenates array inputs into a single array.
 
+## ARRAY_OF_DOUBLES_SKETCH
+
+`ARRAY_OF_DOUBLES_SKETCH(expr, [nominalEntries])`
+
+`ARRAY_OF_DOUBLES_SKETCH(dimensionColumnExpr, metricColumnExpr, ..., [nominalEntries])`
+
+**Function type:** [Aggregation](sql-aggregations.md)
+
+Creates a Tuple sketch.
+
+## ARRAY_OF_DOUBLES_SKETCH_INTERSECT
```

Comment:
@frankgrimes97 two comments:

1) can you arrange the new functions in alphabetical order? That is, 
* ARRAY_OF_DOUBLES_SKETCH_INTERSECT
* ARRAY_OF_DOUBLES_SKETCH_METRICS_SUM_ESTIMATE
* ARRAY_OF_DOUBLES_SKETCH_NOT
* ARRAY_OF_DOUBLES_SKETCH_UNION

2) Make sure a description is included for INTERSECT, NOT, UNION